client/galaxy/scripts/mvc/workflow/workflow.js

```diff
@@ -52,6 +53,30 @@ define( [ 'utils/utils' ], function( Utils ) {
             });
         },
 
+        // Save the workflow as an item in Tool panel
+        register_show_tool_menu: function( self ) {
+            var $el_checkboxes = self.$el.find( '.show-in-tool-panel' );
```

Comment:
Here you can just write `this.$( '.show-in-tool-panel' )`. `el.find` is not necessary.